python/test/unit/runtime/test_autotuner.py

```diff
@@ -43,6 +43,51 @@ def _kernel(src, N, BLOCK_SIZE: tl.constexpr):
     triton.testing.assert_close(src, torch.ones_like(src))
 
 
+def test_hooks():
```

Comment:
Is there documentation for the hooks that we should update so that people know about the `exception` argument?